src/agents/interaction.jl

```diff
@@ -71,18 +65,15 @@ function add_controls!(fig, abmobs, spu)
         s = 0 # reset step counter
         Agents.step!(model[], agent_step!, model_step!, s)
     end
-
     # Clear button
     clear = Button(fig, label = "clear\ndata")
     on(clear.clicks) do c
         abmobs.s[] = 0
         init_dataframes!(model[], adata, mdata, adf, mdf)
         collect_data!(model[], when, adata, mdata, adf, mdf, abmobs.s[])
     end
-
     # Layout buttons
-    controllayout[3, :][:, 1] = MakieLayout.hbox!(step, run; tellwidth = false)
-    controllayout[3, :][:, 2] = MakieLayout.hbox!(reset, clear; tellwidth = false)
+    controllayout[2, :] = MakieLayout.hbox!(step, run, reset, clear; tellwidth = false)
```

Comment:
I think it was nicer to layout the buttons on the previously used 1x2 grid as before. What's the rationale to change all buttons to be in one line?

Reply:
I've made the other two buttons also have two rows for symmetry, and now there is no gap between:

![image](https://user-images.githubusercontent.com/19669089/174597619-c9df55ff-1b60-4750-b801-eeef65b800d7.png)

![image](https://user-images.githubusercontent.com/19669089/174597671-6d06b286-20fb-4097-996d-b5bec13ee73d.png)

I think it looks better this way. The gap before was very large in the second case.

Reply:
I mean, it looks totally OK now. Symmetry is always nice in general. I just preferred that the step/run buttons were clearly separated from reset/clear/update buttons. Having them so close to each other and with more text on them makes it harder to separate them from each other. Doesn't matter too much in the end, probably just my personal taste, so all good.